docs/release_notes.rst

```diff
@@ -6,6 +6,27 @@ PUDL Release Notes
 v2026.X.x (2026-XX-XX)
 ---------------------------------------------------------------------------------------
 
+Application, not Library
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+From now on we will treat PUDL like an application rather than a library, since that's
+what it has become for all intents and purposes. There's no change in the licensing or
+openness of the project -- this is just a technical evolution, based on our experience
+that almost all users want the data, not the pipeline. Treating PUDL like a standalone
```

Comment:
Hmm... I suspect that some users seem to use forked versions of our code to do their own custom transformations (@grgmiller maybe?) we should probably make sure this doesn't break their workflows.